snippets/setup.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import io
+import os
+import sys
+import re
+from setuptools import setup, find_packages
+from distutils.command.build_clib import build_clib
+from distutils.command.install_lib import install_lib
+from distutils import log
+
+NAME = "pyaloha"
+DESCRIPTION = "Alohalytics statistics engine python parser"
+URL = "https://github.com/mapsme/Alohalytics"
+EMAIL = "me@alex.bio"
+AUTHOR = "Alexander Zolotarev"
```

Comment:
haha, you wish!

Reply:
Huh? Alexander Borsuk :)

Reply:
Copied from LICENSE. Have no rights to change it.
PR for changed name if you wish.